ansible/library/port_alias.py

```diff
@@ -7,6 +7,7 @@
 from operator import itemgetter
 from itertools import groupby
 from collections import defaultdict
+from devutil import conn_graph_helper
```

Comment:
@bingwang-ms I don't think conn_graph_helper is designed to be used by ansible library, do you have concern here?

Maybe a better way would be read port_config.ini, and call libraray/conn_graph_facts to get connection graph facts, then let the template decide which one to use?

Reply:
Agree. I suggest that ```devutil``` not be referenced directly from ```Ansible``` library.